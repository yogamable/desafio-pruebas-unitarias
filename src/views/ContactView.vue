<template>
    <div>
        <h1>Contacto</h1>
    </div>
</template>

<script>
export default {
    name: 'ContactView',
}
</script>

<style scoped>

</style>