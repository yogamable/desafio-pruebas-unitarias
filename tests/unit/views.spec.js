import { mount } from '@vue/test-utils';
import AboutView from '@/views/AboutView.vue';
import ContactView from '@/views/ContactView.vue';

describe('Test de vista About', () => {
  it('Test 1: Matchea el snapshot de la vista About correctamente', () => {
    const wrapper = mount(AboutView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});

describe('Test de vista Contact', ()=>{
  it('Test 2: Matchea el snapshot de la vista Contact correctamente', () => {
    const wrapper = mount(ContactView);
    expect(wrapper.html()).toMatchSnapshot();
  });
})
