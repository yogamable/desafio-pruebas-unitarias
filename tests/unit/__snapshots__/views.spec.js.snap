// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test de vista About Match de snapshot con AboutView.vue 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;

exports[`Test de vista Contact Match de snapshot con ContactView.vue 1`] = `
<div>
  <h1>Contacto</h1>
</div>
`;
